tests/unit/completion/test_models.py

```diff
@@ -23,9 +23,11 @@
 import random
 import string
 from datetime import datetime
+from unittest import mock
 
 import pytest
 from PyQt5.QtCore import QUrl
+from PyQt5.QtWebEngineWidgets import QWebEngineHistory, QWebEngineHistoryItem
```

Comment:
This [breaks the tests](https://travis-ci.org/qutebrowser/qutebrowser/jobs/427092466) with QtWebKit (without QtWebEngine installed). I guess you could catch the `ImportError` and ignore it here, and in the tests do `pytest.importorskip('PyQt5.QtWebEngineWidgets')`.